webapp/src/components/SampleTable.vue

```diff
@@ -88,8 +91,8 @@ export default {
     this.getSamples();
   },
   methods: {
-    getSamples() {
-      getSampleList();
+    async getSamples() {
+      await getSampleList();
```

Comment:
```suggestion
    getSamples() {
      getSampleList();
```

I think we can remove this now